src/components/AddItems.js

```diff
@@ -19,38 +26,44 @@ const AddItemsToList = () => {
   function submitShoppingListItemHandler(event) {
     event.preventDefault();
 
-    const userToken = localStorage.getItem('token');
-
     if (shoppingListItemName === '') {
       alert('Please enter item name...');
       return;
     }
 
-    const values = {
+    const item = {
```

Comment:
I like that you've changed this to item as I think it makes more sense, we can also change this in our branch so that we are consistent.